src/utils/queryServer.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+export function queryServer(path, params = undefined) {
+  return new Promise((resolve, reject) => {
+    const baseURL = 'http://127.0.0.1:8000';
```

Comment:
Sorry for my lack of knowledge in React if I am reading this incorrectly, but is the baseURL is hardcoded at 127.0.0.1?